packages/connect-express/conformance/server.ts

```diff
@@ -97,8 +97,10 @@ function main() {
   switch (req.httpVersion) {
     case HTTPVersion.HTTP_VERSION_1:
       server = req.useTls
-        ? https.createServer(serverOptions, app)
-        : http.createServer(app);
+        ? // eslint-disable-next-line @typescript-eslint/no-misused-promises -- typing issue in express
```

Comment:
Not sure why eslint considers this to be a promise misuse. Either way, it isn't something we can fix, since it's an issue with Express' typings.